module/VuFind/src/VuFind/Db/Service/CommentsServiceInterface.php

```diff
@@ -111,6 +121,23 @@ public function getStatistics(): array;
      */
     public function getCommentById(int $id): ?CommentsEntityInterface;
 
+    /**
+     * Get a paginated result of all comments by user id
```

Comment:
Add punctuation for consistency.

```suggestion
     * Get a paginated result of all comments by user id.
```